src/sentry/static/sentry/app/components/dropdownButton.tsx

```diff
@@ -1,26 +1,27 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import styled from '@emotion/styled';
 
 import Button from 'app/components/button';
 import InlineSvg from 'app/components/inlineSvg';
 
-class DropdownButton extends React.Component {
-  static propTypes = {
-    isOpen: PropTypes.bool,
-    showChevron: PropTypes.bool,
-    forwardRef: PropTypes.any,
-  };
-  render() {
-    const {isOpen, showChevron, children, forwardRef, ...otherProps} = this.props;
-    return (
-      <StyledButton type="button" isOpen={isOpen} ref={forwardRef} {...otherProps}>
-        {children}
-        {showChevron && <StyledChevronDown />}
-      </StyledButton>
-    );
-  }
-}
+type Props = React.ComponentProps<typeof Button> & {
+  isOpen?: boolean;
+  showChevron?: boolean;
+  forwardedRef?: React.Ref<typeof Button>;
+};
+
+const DropdownButton = ({
+  isOpen,
+  showChevron = false,
```

Comment:
Should parameters with default values come at the end of the destucturing list?

Reply:
No preference for me. 

Might be worth discussing in the next TSC.

Reply:
I'm indifferent, I think I preserved order here. Important to note, this is a destructured object.

Reply:
Moved it to the end 🤷‍♂